internal/controller/integrationpipeline/integrationpipeline_adapter_test.go

```diff
@@ -673,55 +671,6 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
 			}
 			Expect(k8sClient.Update(ctx, hasEnv)).Should(Succeed())
 		})
-		It("ensures ephemeral environment is deleted for the given pipelineRun ", func() {
```

Comment:
I think this entire `When` block can be removed since it just contains `BeforeEach` block now.

Reply:
Agreed - updated [here](https://github.com/konflux-ci/integration-service/pull/705/files#diff-e76f3c82a5981d291c11d7bb74345c04009bea89749b5dd1ae3be21791d41c13)